<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogBeau</class>
 <widget class="QDialog" name="DialogBeau">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_explain_pot_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>vs RHE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_step">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Step: (usually 0.01 V)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDial" name="dial_temp">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>50</width>
     <height>64</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_max_Pot">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Maximum Potential (V)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_maxpot">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_particle_counter">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>37</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_min_Pot">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Minimum Potential (V)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_minpot">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>37</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_reaction">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_red_curr">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Reduction Current</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_temp_display">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>37</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_temp">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Temperature (K)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>530</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Start Computation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_maxpot">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>37</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_minpot">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_reaction">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select Reaction:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_particles_added">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Particles added: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_step">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.01</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_explain_pot">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>vs RHE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Select a particle to add</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_ActvsPot">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Activity vs Potential</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButtonActVsSize">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>140</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Activity vs Size</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="checkBox_corrections">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>171</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>I wish to apply corrections</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ZPE">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_TDS">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_coeff">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>290</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ZPE">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>230</y>
     <width>37</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Δ ZPE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_TDS">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>260</y>
     <width>37</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TΔS </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_coeff">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>290</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Coefficient</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_filename_out">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>440</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_fileout">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>440</y>
     <width>21</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>. . .</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_outputfile">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Please select the folder where you wish any output file(s) to be stored.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
